readme.md

```diff
@@ -7,28 +7,28 @@ The page should match the design Pixel Perfect: all the sizes, colors and distan
 
 ## Requirements:
 
-- reset browser default margins.
-- use [Roboto font](https://fonts.google.com/specimen/Roboto)
-- use semantic tags: `<header>`, `<img>`, `<nav>`, `<ul>`, `<li>` and `<a>`
-- the header should stretch the full page width
-- the height should be set for nav links (not the header), take it from the design.
-- header content should be vertically centered.
-- Logo should also be a link with an image inside (from [src/images](src/images)). But it should not be a part of the `<nav>`.
-- ❗️ the blue link with a line below should have a class `is-active` in addition to any other classes you add.
-- ❗️ add `data-qa="hover"` attribute to the 4th link for testing (`Laptops & computers`)
-- link color should be changed on `:hover`
+- reset browser default margins. DONE
+- use [Roboto font](https://fonts.google.com/specimen/Roboto) DONE
+- use semantic tags: `<header>`, `<img>`, `<nav>`, `<ul>`, `<li>` and `<a>` DONE
+- the header should stretch the full page width DONE
+- the height should be set for nav links (not the header), take it from the design. DONE
+- header content should be vertically centered. DONE
+- Logo should also be a link with an image inside (from [src/images](src/images)). But it should not be a part of the `<nav>`. DONE
+- ❗️ the blue link with a line below should have a class `is-active` in addition to any other classes you add. DONE
+- ❗️ add `data-qa="hover"` attribute to the 4th link for testing (`Laptops & computers`) DONE
+- link color should be changed on `:hover` DONE
 - Use the `::after` and position it relative to a link with `is-active` class
-- Don't use flex `gap` property for indents. It's not supported in tests, use `margin` instead.
-- There should not be margins before the first and after the last list items
-- Don't just copy all styles from Figma. Think, which of them are relevant. Uneven sizes (e.g. `line-height: 14.6px`) are definitely useless.
-- Nav Links should not have any padding, but have text centered
+- Don't use flex `gap` property for indents. It's not supported in tests, use `margin` instead. DONE
+- There should not be margins before the first and after the last list items DONE
+- Don't just copy all styles from Figma. Think, which of them are relevant. Uneven sizes (e.g. `line-height: 14.6px`) are definitely useless.DONE
+- Nav Links should not have any padding, but have text centered DONE
 
 ## Checklist
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://2pacanowski.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://2pacanowski.github.io/layout_moyo-header/report/html_report/)
```

Comment:
Don't forget to add these links to your pull request description
![image](https://github.com/mate-academy/layout_moyo-header/assets/80700247/3a14f36e-2bdc-4ccd-a981-c3accce0012e)

Reply:
@nazarmatsevych links added